acquia/blt
# Fixes #3357: Don't use relative path for memcache PSR-4 path registration

Fixes #3357 
--------

Changes proposed:
---------
(What are you proposing we change?)

- Use absolute path during memcache PSR-4 class path registration

Steps to replicate the issue:
----------
(If this PR is not fixing a defect/bug, you can remove this section.)

1. Use Drupal VM with Drush Lanucher installed and Memcache set up for Drupal
2. Run `drush cr` and see the PHP fatal error message described in #3357

Steps to verify the solution:
-----------
(How does someone verify that this does what you think does?)

1. Use Drupal VM with Drush Lanucher installed and Memcache set up for Drupal
2. Run `drush cr` and see if error mentioned in #3357 is still thrown
